repos/rust-lang/lang-team.toml

```diff
@@ -0,0 +1,7 @@
+org = "rust-lang"
+name = "lang-team"
+description = "Home of the Rust lang team"
+bots = ["rustbot"]
+
+[access.teams]
+lang = "maintain"
```

Comment:
Release also has write permissions currently cc @Mark-Simulacrum

Reply:
let's also add the various lang subteams:

* ops
* docs
* types

Reply:
All with the maintain permission? Or just read/write?